subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/InputArtifactDependencies.java

```diff
@@ -29,8 +29,30 @@
 /**
  * Attached to a property that should receive the <em>artifact dependencies</em> of the {@link InputArtifact} of an artifact transform.
  *
- * The order of the files match that of the dependencies in the source artifact view.
- * The type of the injected dependencies is {@link org.gradle.api.file.FileCollection}.
+ * <p>
+ *     The order of the files match that of the dependencies in the source artifact view.
+ *     The type of the injected dependencies is {@link org.gradle.api.file.FileCollection}.
```

Comment:
Perhaps clarify that the corresponding property must be declared as type `FileCollection`.

Reply:
Fixed in https://github.com/gradle/gradle/pull/9525.